webapp/src/pages/Settings.tsx

```diff
@@ -92,6 +102,11 @@ const SUPPORTED_SPACY_MODELS: SupportedSpacyModels[] = [
   "fr_core_news_md",
 ];
 const USE_CUDA_OPTIONS = ["auto", "true", "false"] as const;
+const POSTPROCESSORS_CLASS_NAMES: string[] = [
```

Comment:
I would suggest making the type more robust in case the values need to change:
```suggestion
type PostprocessorClassName = (TemperatureScaling | ThresholdConfig)["class_name"];
const POSTPROCESSORS_CLASS_NAMES: PostprocessorClassName[] = [
```
with the types in `src/types/api.ts` of course:
```ts
type TemperatureScaling = components["schemas"]["TemperatureScaling"];
type ThresholdConfig = components["schemas"]["ThresholdConfig"];
```